alma/alma-8.x/common/disable_user_namespaces.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "user.max_user_namespaces=0" > /etc/sysctl.d/userns.conf
```

Comment:
When this file was added, were the execute permissions added?

Reply:
No.  I just checked and this file returns 100644.  I'm fixing this now (755) and will include in the next commit for this PR.